SU-SWS/Stanford-Drupal-Profile
# SITES-1101: BEAN title handling that is SMRT.

# READY FOR REVIEW

# Summary
- Handles the new bean->title variable in the bean template more elegantly by creating new logic around the way blocks and beans create their. 

# Needed By (Date)
- ?

# Urgency
- Semi urgeant 

# Steps to Test

- Install stanford profile
- Enable bean module 1.11
- Create a new bean type and add a body field
- Create a new bean and place it into the left sidebar filling in each of the bean entity fields
- See no duplicate title
- Check out bean 1.13
- Clear cache and revisit page with the bean on it
- See no duplicate title and that the bean title is the BEAN's title not the blocks
- Edit the bean type's view modes
- Move the title field to hidden and save
- Clear all caches
- Review the page with the bean on it again and see that the title is now owned by the block and not the bean
- Edit the block title configuration and save
- Refresh page and see new block title and not the bean title.

*Optional:*
- Play around with setting and not setting the block title to `<none>`
- Play around with setting and not setting content in to the title field on beans
- Play around with other view modes

# Affected Projects or Products
- Everything?

# Associated Issues and/or People
- https://www.drupal.org/project/bean/issues/2996282 
- SITES-1101

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
